server/venueless/settings.py

```diff
@@ -5,6 +5,8 @@
 
 from django.utils.crypto import get_random_string
 from kombu import Queue
+from redis.asyncio.retry import Retry
```

Comment:
The Redis client used in channel layer is non-async one, you should use `Retry` from non-async module also.

Reply:
Ah no, my mistake.